android/java/org/chromium/chrome/browser/settings/developer/BraveQAPreferences.java

```diff
@@ -71,12 +84,40 @@ public void onCreate(Bundle savedInstanceState) {
         if(mDebugNTP != null) {
             mDebugNTP.setOnPreferenceChangeListener(this);
         }
+
+        mDbUtil = BraveDbUtil.getInstance();
+
+        mImportRewardsDb = findPreference(QA_IMPORT_REWARDS_DB);
+        mExportRewardsDb = findPreference(QA_EXPORT_REWARDS_DB);
+        setRewardsDbClickListeners();
+        checkQACode();
```

Comment:
Moved here for proper `ACTION_GET_CONTENT` handling. When it is `onStart` it request code additionaly, when file is chosen.